.github/workflows/build-example-web.yml

```
@@ -6,58 +6,25 @@ env:
   COMMIT_SHORT_SHA: ${{ github.sha }}
 
 jobs:
-  # 基础构建 job - 处理共同的依赖安装和包构建
-  build_base:
+  build_expo:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)